Makefile.common

```diff
@@ -106,7 +106,7 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint yamllint unused build test
+common-all: style check_license lint yamllint unused build test
```

Comment:
Doesn't this invalidate your assertion that `precheck` was unused?

Reply:
Maybe unused isn't the right term. It's more that there aren't any prechecks as far as I can tell, the target is effectively a noop

Reply:
The makefile.common contains targets that can be overriden in Makefile and it is shared in all projets. What is noop here can be an op in other projects.

Reply:
See [this query](https://github.com/search?q=%28org%3Aprometheus-community+OR+org%3Aprometheus%29+AND+%28precheck+OR+common-staticcheck%29&type=code): I haven't been able to find a single place where this was actually used? I may be wrong though, but considering that there is no place in both those orgs, I'd expect this to effectively not be used anywhere